app/templates/components/modals/add-user-role-modal.hbs

```diff
@@ -0,0 +1,48 @@
+<div class="header">
+  {{t 'Add Role'}}
+</div>
+<div class="content">
+  <form class="ui form" autocomplete="off" {{action (optional submit) on='submit' preventDefault=true}}>
+    <div class="field">
+      <label class="required">
+        {{t 'User Email'}}
+      </label>
+      {{input type='text' name='user_email'}}
+    </div>
+    <div class="field">
+      <label class="required">
+        {{t 'User role'}}
+      </label>
+      {{#ui-dropdown class='fluid selection'}}
+        {{input type='hidden' name='user_role'}}
+        <i class="dropdown icon"></i>
+        <div class="default text">
+          {{t 'Select a role'}}
+        </div>
+        <div class="menu">
+          <div class="item" data-value="track_organizer">
+            {{t 'Track Organizer'}}
+          </div>
+          <div class="item" data-value="coorganizer">
+            {{t 'Co-organizer'}}
+          </div>
+          <div class="item" data-value="moderator">
+            {{t 'Moderator'}}
+          </div>
+          <div class="item" data-value="registrar">
+            {{t 'Registrar'}}
+          </div>
+        </div>
+      {{/ui-dropdown}}
+    </div>
+  </form>
+</div>
+<div class="actions">
+  <button type="button" class="ui black button" {{action 'close'}}>
+    {{t 'Cancel'}}
+  </button>
+  <button type="button" class="ui green right labeled icon button" {{action 'addRole'}}>
```

Comment:
@niranjan94 Wrapping it in the form element inside the element, breaks the default look of the `modal`. `content` and `actions` div class need to be separate, enclosing in a form tag, it interferes with semantic UI's modal layout.  and giving `form` as a tagName to the modal component  is not a good idea as @geekyD pointed out in a previous PR, it causes loss of validation.
Is it acceptable if I rather including in `actions` class, use buttons defined by me and make them _feel_ similar to the modal's default buttons?